pkg/cmd/ports/forward.go

```diff
@@ -116,3 +119,11 @@ func ForwardPublicPort(workspaceId, projectName string, hostPort, targetPort uin
 
 	return service.Run(context.Background())
 }
+
+func RenderQr(s string, inverseColor bool) {
```

Comment:
We should either change this to a private function or move it to views/util/qr.go

Reply:
Changed to private function